modules/osd-monitoring-default-monitoring-components.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-understanding-the-monitoring-stack.adoc
+
+[id="default-monitoring-components_{context}"]
+= Default monitoring components
+
+By default, the {product-title} monitoring stack includes these components:
+
+.Default monitoring stack components
+[options="header"]
+|===
+
+|Component|Description
+
+|Cluster Monitoring Operator
+|The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+
+|Prometheus Operator
+|The Prometheus Operator (PO) in the `openshift-monitoring` project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
+
+|Prometheus
+|Prometheus is the monitoring system on which the {product-title} monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+
+|Prometheus Adapter
+|The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` and `oc adm top pods` commands.
+
+|Alertmanager
+|The Alertmanager service handles alerts received from Prometheus. Alertmanager is also responsible for sending the alerts to external notification systems.
+
+|`kube-state-metrics` agent
```

Comment:
change from
``kube-state-metrics``
to 
`kube-state-metrics`

Reply:
This is the correct formatting - they all use backticks (`) in the text editor. It renders correctly - please see https://deploy-preview-30483--osdocs.netlify.app/openshift-rosa/latest/monitoring/osd-understanding-the-monitoring-stack.html